spec/system/provider_interface/provider_makes_changes_before_offer_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'rails_helper'
+
+RSpec.feature 'Provider makes changes before making an offer' do
+  include CourseOptionHelpers
+  include DfESignInHelpers
+
+  scenario 'Provider makes changes to course and location' do
+    given_i_am_a_provider_user_with_dfe_sign_in
+    when_change_response_feature_is_activated
+    and_i_am_permitted_to_see_applications_for_two_providers
+    and_an_application_choice_exists_for_one_of_my_providers
+    and_another_two_course_options_exist_for_this_provider
+    and_i_sign_in_to_the_provider_interface
+    and_i_view_an_application
+
+    and_i_click_on_respond_to_application
+    then_i_see_options_to_make_an_offer
+
+    when_i_choose_make_offer_but_change_course
+    then_i_see_all_courses_for_this_provider
+    and_i_can_change_the_course
+    and_i_see_all_available_locations_for_this_course
+    and_i_can_change_the_location
```

Comment:
Should we also cover changing to offer to the other provider?

Reply:
@duncanjbrown https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1667/commits/7df8f55e19bb157cb69da9e4ca55daa098b66f4d